acquia/acquia_cms
# ACMS-779: Refactor acquia_cms_site_studio & acquia_cms_common module

**Motivation**

Fixes #[ACMS-779](https://backlog.acquia.com/browse/ACMS-779)

**Proposed changes**

Update modules dependencies so that it can be installed independently 

**Testing steps**

Install vanilla drupal with standard profile and install each acquia cms module and check

**Merge requirements**
